src/views/AvailableTeams.vue

```diff
@@ -15,6 +10,7 @@
           {{ team.name }}
         </p>
       </router-link>
+      <h3>Available teams</h3>
```

Comment:
? There's an h1 below for available teams

Reply:
Where do you see that? I don't.

Reply:
Line 31 / 27

Reply:
thanks. Took it out